app/scripts/templates/permissions.mustache

```diff
@@ -0,0 +1,30 @@
+<div id="main-content" class="card">
+  <header>
+    <h1 id="fxa-confirm-header">{{#t}}%(serviceName)s would like to know...{{/t}}</h1>
+  </header>
+
+  <section>
+    <div class="error"></div>
+    <div class="success"></div>
+
+    <div id="permissions">
+      <p>
+        <span>{{#t}}Email{{/t}}</span>
+        <span class="email">{{ email }}</span>
+      </p>
+    </div>
+
+    <div class="privacy-links">
+      {{#t}}By proceeding, I agree to the <a id="fxa-tos" href="%(termsUri)s">Terms of Service</a> and <a id="fxa-pp" href="%(privacyUri)s">Privacy Notice</a> of %(serviceName)s (%(serviceUri)s).{{/t}}
+    </div>
+
+    <div class="button-row">
+      <a id="proceed" class="button" href="#">{{#t}}Proceed{{/t}}</a>
```

Comment:
Not a normal [submit button](https://github.com/mozilla/fxa-content-server/blob/cdd999e395d560f67762c3b3d258859e5852902a/app/scripts/templates/sign_in.mustache#L81) like the other templates?